addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -69,149 +64,136 @@
+            <h3>Innovation <br class="d-none d-lg-inline"/>Hub</h3>
+            <p>Where ideas come to life</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 6 / 1 / 11 / 4; z-index: 2;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item g-col-lg-3 text-center o_cc o_cc2 o_colored_level g-height-4 col-lg-3 rounded-4" data-name="Block" style="grid-area: 6 / 1 / 10 / 4; z-index: 2;">
+            <h3>Community <br class="d-none d-lg-inline"/>Focus</h3>
+            <p>Building connections</p>
         </div>
-
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 4 / 11 / 10; z-index: 3;">
-            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item g-col-lg-6 o_grid_item_image text-center o_colored_level g-height-9 col-lg-6" data-name="Block" style="grid-area: 1 / 4 / 10 / 10; z-index: 3;">
+            <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto rounded-4" alt=""/>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 1 / 10 / 6 / 13; z-index: 4;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item g-col-lg-3 text-center o_cc o_cc2 o_colored_level g-height-4 col-lg-3 rounded-4" data-name="Block" style="grid-area: 1 / 10 / 5 / 13; z-index: 4;">
+            <h3>Key <br class="d-none d-lg-inline"/>Milestone</h3>
+            <p>Reaching new heights together</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 6 / 10 / 11 / 13; z-index: 5;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item g-col-lg-3 text-center o_cc o_cc4 o_colored_level g-height-5 col-lg-3 rounded-4" data-name="Block" style="grid-area: 5 / 10 / 10 / 13; z-index: 5; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-1) 100%);">
+            <h3>Greater <br class="d-none d-lg-inline"/>Impact</h3>
+            <p>Making a difference every day</p>
         </div>
     </div>
 </template>
 
 <template id="s_masonry_block_alternation_text_template" groups="base.group_user">
-    <div class="row o_grid_mode" data-row-count="3">
+    <div class="row o_grid_mode" data-row-count="3" style="gap: 0 16px;">
         <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc2" data-name="Block"
```

Comment:
Same vertical issue, the "centered" button in the WB is not selected.
Add `justify-content-center`

Same for the other blocks of this template ;)